content_sync/utils.py

```diff
@@ -0,0 +1,38 @@
+"""Content sync utility functionality"""
+import logging
+import os
+from typing import Optional
+
+from websites.constants import WEBSITE_CONTENT_FILETYPE
+from websites.models import WebsiteContent
+from websites.site_config_api import SiteConfig
+
+
+log = logging.getLogger()
+
+
+def get_destination_filepath(
```

Comment:
I moved this because it's not Github-specific. When I moved it to `content_sync/api.py` where it probably belongs, I ran into circular import issues between `api.py` and `tasks.py`. To avoid a larger refactor I just put the function here in `utils.py`